schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE vet_clinic.animals (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(8, 2)
 );
```

Comment:
- [ ] Great job on your progress in writing your queries! 🚀 However, I noticed that there's a requirement specified in the project guidelines that I didn't see implemented in your code. 

    - The requirement is to add a new column named `species` with a data type of `string` to your animal's table. This column is essential for the project, and I'd like to encourage you to consider adding the `species` column to your animal table. See below for more details on the project requirements:
    
<img width="778" alt="Screenshot 2023-11-09 at 21 13 04" src="https://github.com/BHObserver/Vet-Clinic-Database/assets/107192076/41464466-a423-4d7b-8119-1919d15ec9d8">

    
🐾 It's a crucial part of the project and will help you meet the project's objectives. 
Happy coding! 👍😊